storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -7790,8 +7790,10 @@ xtrabackup_prepare_func(void)
 	}
 
 	/* print the binary log position  */
-	trx_sys_print_mysql_binlog_offset();
```

Comment:
Should these lines be completely removed instead?

Reply:
Commented this line and tested several times.
No issues found. So yes, it can be removed instead.

Reply:
Removed specified line and added test case